client/plots/disco/events.js

```diff
@@ -0,0 +1,283 @@
+import { tooltip } from '#src/client'
```

Comment:
see if you can replace tooltip with `import {Menu} from '#dom/menu`

Reply:
I've tried to replace it with Menu, but the tooltip has additional methods like: `getPosChange`. I would say we keep it for now and I'll remove it when I refactor the whole disco plot. I can add a TODO for now.

Reply:
Disco plot-related code is extracted to the helper.js script in the disco folder.